package-lock.json

```diff
@@ -8,7 +8,6 @@
       "version": "7.10.4",
       "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.10.4.tgz",
       "integrity": "sha512-vG6SvB6oYEhvgisZNFRmRCUkLz11c7rp+tbNTynGqc6mS1d5ATd/sGyV6W0KZZnXRKMTzZDRgQT3Ou9jhpAfUg==",
-      "dev": true,
```

Comment:
The `package-lock.json` has some changes because some packages are optional or marked as dev on different OS (Windows/Mac/Linux). Thus, `npm install` will update this file without useful changes. 

Read more: https://github.com/npm/npm/issues/17722

To avoid this, make sure to not include this file in your commits or install dependencies with `npm install --no-save`.

To revert this change, you can use `git rm --cached` command.